.circleci/config.yml

```diff
@@ -108,28 +75,26 @@ workflows:
   version: 2.1
   ci-cd:
     jobs:
-      - install
+      - install:
+          context: org-global
+      - compatibility:
+          context: org-global
+          matrix:
+            parameters:
+              image:
+                - "14.21"
+                - "16.20"
+                # - "18.17" // default image
```

Comment:
Why not to remove the step "test", remove this comment and rename step "compatibility" to "test"? It won't be clearer? It is to avoid an `npm i`?